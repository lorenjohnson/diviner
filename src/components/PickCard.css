.pickCard {
  width: 100px;
  height: 400px;
  padding: 20px;
  background-color: white;
}
