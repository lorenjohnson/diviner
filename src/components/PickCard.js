import React from 'react'
import './PickCard.css'

export function PickCard () {
  return (
    <div className='pickCard'>
      Diviner it's time to pick a card
    </div>
  )
}

export default PickCard
