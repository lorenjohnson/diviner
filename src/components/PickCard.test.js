import React from 'react'
import ReactTestRenderer from 'react-test-renderer'
import PickCard from './PickCard'

it('matches last snapshot', () => {
  const component = ReactTestRenderer.create(<PickCard />)
  expect(component.toJSON()).toMatchSnapshot()
})
