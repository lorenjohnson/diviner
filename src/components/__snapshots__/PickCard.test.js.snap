// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches last snapshot 1`] = `
<div
  className="pickCard"
>
  Diviner it's time to pick a card
</div>
`;
